omnibus/config/software/lmdb.rb

```diff
@@ -0,0 +1,25 @@
+name "lmdb"
+default_version "0.9.29"
+
+source :url => "https://github.com/LMDB/lmdb/archive/LMDB_#{version}.tar.gz",
+       :sha256 => "22054926b426c66d8f2bc22071365df6e35f3aacf19ad943bc6167d4cae3bebb",
+       :extract => :seven_zip
+
+relative_path "libraries/liblmdb"
```

Comment:
I think this should be:
```suggestion
relative_path "lmdb-LMDB_#{version}/libraries/liblmdb"
```

since expanding the `LMDB_0.9.29.tar.gz` archive puts the contents in a directory named `lmdb-LMDB_0.9.29`.

Reply:
Ah gotcha, thanks! I'll try this.